package.json

```diff
@@ -46,11 +45,12 @@
     "mocha": "^8.1.2",
     "rc": "^1.0.0",
     "should": "^13.2.3",
-    "sinon": "^9.0.2"
+    "sinon": "^9.0.2",
+    "@commitlint/config-conventional": "^12.1.4"
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/strongloop/loopback-connector-postgresql.git"
+    "url": "https://github.com/loopbackio/loopback-connector-postgresql.git"
   },
   "license": "Artistic-2.0",
```

Comment:
Should it be MIT ?

Reply:
I didn't change the license because I'm not sure what's the implication of it. For example, for existing users who accepted the license term for using this module under the Artistic-2.0 license, what would it mean to them if license changed. 
IIUC (I'm not license expert so I could be wrong), `Artistic-2.0` and `MIT` are similar license.